### PR TITLE
Open User register and login

### PR DESCRIPTION
**Pull request by RainiWijewantha**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -62,7 +62,15 @@
 				<groupId>org.springframework.boot</groupId>
 				<artifactId>spring-boot-maven-plugin</artifactId>
 			</plugin>
-		</plugins>
+            <plugin>
```
Comment (kavicastelo) [inline pom.xml:65]: this is an unnecessary part.
if you need to add this, provide me a reason

**Review by kavicastelo — CHANGES_REQUESTED** [review]
make change the requested changes

File: pom.xml (hunk 1 above)
Comment (RainiWijewantha) [reply pom.xml:65]: It happens with my IDE. I will try to solve this problem immediately.

**Review by kavicastelo — APPROVED** [review]
reviewed and approved

File: pom.xml (hunk 1 above)
Comment (kavicastelo) [reply pom.xml]: use compatible java versions

**Pull request merged**